### ISSUE TITLE
Problem to follow Cloud Firestore Web Codelab Tutorial, page 2, step 3

### ISSUE THREAD

**Issue by Ilya-Lad** [firebase/friendlyeats-web#92]
Windows 10, modern Chrome.
After selecting **Start in test mode**, a blue button **Next**  is shown on the form (Instead of the button **Enable** shown in the tutorial). Clicking the **Next** button shows the form for selecting Cloud Firestore location, with num5(us-central) being selected by default. That location is appropriate in my case. The blue button **Done** is shown in the right bottom of the form. Clicking the button cause a spinning cursor for about a minute, and then causes an error:

"Cannot enable Firestore for this project
An unknown error occurred"

(Attempt to follow the tutorial farther shows that the mock data is not added to the DB).

**Comment by Ilya-Lad** [comment]
Today the problem has gone. I looked in the status info. Yesterday they had issues. Not sure what that all means. But I close the issue.

**Comment by amreldsokey** [comment]
> Today the problem has gone. I looked in the status info. Yesterday they had issues. Not sure what that all means. But I close the issue.

how you solve it ?

**Comment by Ilya-Lad** [comment]
On the day it was the problem, the Firebase status indicated issues. My problem gone on its own the next day, and the Firebase status did not show  issues on that day. So I interpreted it that my problem resulted from the Firebase  issues.
    On Monday, June 8, 2020, 08:33:01 AM EDT, amreldsokey <notifications@github.com> wrote:  
 
 



Today the problem has gone. I looked in the status info. Yesterday they had issues. Not sure what that all means. But I close the issue.


how you solve it ?
